docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+
+These steps use placeholder values for the following: `PROJECT_ID`, `SERVICE_ACCOUNT_NAME`. (**GWH_YS02 where do they get these values from? Is it just their choice**)
+
+1. Click **Active Cloud shell**.
+
+2. Switch to your project:
+    ```
+    $ gcloud config set project {PROJECT_ID}
+    ```
+ 
+3. Enable required APIs:
+    ```
+    $ gcloud services enable compute.googleapis.com confidentialcomputing.googleapis.com
+    ```
+
+4. Create a service account to run the <a href="#gl-workload">workload</a>:
+    ```
+    $ gcloud iam service-accounts create {SERVICE_ACCOUNT_NAME}
+    ```
+
+5. Grant the required permission, `confidentialcomputing.workloadUser`, to the service account. This permission grants the ability to generate an attestation token and run a workload in a VM:
+    ```
+    $ gcloud projects add-iam-policy-binding {PROJECT_ID} \
+      --member=serviceAccount:{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com \
+      --role=roles/confidentialcomputing.workloadUser
+    ```
+
+6. (Optional) Grant the optional permission, `logging.logWriter`, to the service account. This permission grants access to write and view logs in debug mode, and is useful for testing:
+    ```
+    $ gcloud projects add-iam-policy-binding {PROJECT_ID} \
+      --member=serviceAccount:{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com \
+      --role=roles/logging.logWriter
+    ```
+  
+7. Add a VPC rule to allow public access on port 8080, the default exposed port for the UID2 operator:
+    ```
+    $ gcloud compute firewall-rules create operator-tcp \
+      --direction=INGRESS --priority=1000 --network=default --action=ALLOW \
+      --rules=tcp:8080 \
+      --source-ranges=0.0.0.0/0 \
+      --target-service-accounts={SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com
+    ```
+
+### UID2 Operator Account Setup
+Ask your UID2 contact to register your organization as a UID2 Operator. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+(**GWH_YS03 what information do they need to provide for registration? In case you don't know, who should I ask?**)
+
+When the registration process is complete, you'll receive the following:
+
+ - `{IMAGE_SHA}`: A valid UID2 operator image digest, used in configuration. For details, see [Integration Deployment](#integration-deployment) and  [Production Deployment](#production-deployment).
+ - `{API_TOKEN}`: An API token, exclusive to you, that identifies you with the UID2 service as a private operator. This value is used in configuration. For details, see [Integration Deployment](#integration-deployment) and  [Production Deployment](#production-deployment).
+
+(**GWH_YS04 what exactly do we send them? Is there more than the above? GEN TO REVIEW THE RECORDING BEFORE ASKING THIS ALSO: do they use the same value for integ and prod? Or do they get separate values? We should be clear about that**)
```

Comment:
From what I heard, we will send to them via email.
We will also provide instructions (commands to bring up VMs) to them, or a link to the instruction. (e.g. the deployment section of the public doc.)

We have different value for integ and prod.

Reply:
Updated, thx.